opal-core-ws/src/main/java/org/obiba/opal/web/identifiers/IdentifiersMappingResource.java

```diff
@@ -196,13 +194,15 @@ public Response importIdentifiers(Magma.DatasourceFactoryDto datasourceFactoryDt
   @POST
   @Path("/_generate")
   public Response importIdentifiers(@QueryParam("type") String entityType, @QueryParam("size") Integer size,
-      @QueryParam("zeros") Boolean zeros, @QueryParam("prefix") String prefix) {
+      @QueryParam("zeros") Boolean zeros, @QueryParam("luhn") Boolean luhn, @QueryParam("prefix") String prefix) {
```

Comment:
Naming the parameter "checksum" in place of "luhn" is les restrictive. Please update everywhere it applies.

Reply:
Changed some parameter names, also changed the check box label to only say "checksum" but mentioned in the description that it is a Luhn check digit.